### PR TITLE
Log requests path with their status

### PR DESCRIPTION
**Pull request by dmathieu**

### CODE REVIEW

File: util/response_logger.go [hunk 1]
```diff
@@ -6,13 +6,15 @@ import (
 	"strconv"
 )
 
-func NewResponseLogger(w http.ResponseWriter, requestID string) *responseLogger {
-	return &responseLogger{ResponseWriter: w, status: http.StatusOK, requestID: requestID}
+// NewResponseLogger creates a new logger for http responses
```
Comment (tt) [inline util/response_logger.go]: "HTTP"

**Comment by tt** [comment]
Nitpick aside, looks good to me.

**Comment by tt** [comment]
This can be another pull request but should we also log the user agent?

**Comment by dmathieu** [comment]
Good idea. I've added it.

**Pull request merged**